src/routes/repository.svelte

```diff
@@ -1,191 +1,72 @@
-    }}>
-    Refresh
-  </button>
-
-  <table>
-    <tr>
-      <th>Repository</th>
-      {#each Object.keys(repostats) as repository}
-        <th>{repository}</th>
-      {/each}
-    </tr>
-
-    <tr>
-      <th>AIP count</th>
-      {#each Object.keys(repostats) as repository}
-        <td>{repostats[repository].count}</td>
-      {/each}
-    </tr>
-
-    <tr>
-      <th>Total disk used</th>
-      {#each Object.keys(repostats) as repository}
-        <td>
-          {repostats[repository].sum} ({filesize(repostats[repository].sum)})
-        </td>
-      {/each}
-    </tr>
-
-    <tr>
-      <th>Earliest validated</th>
-      {#each Object.keys(repostats) as repository}
-        <td>
-          {#if verified[repository] && verified[repository].earliest}
-            {verified[repository].earliest}
-          {/if}
-        </td>
-      {/each}
-    </tr>
-
-    <tr>
-      <th>Latest validated</th>
-      {#each Object.keys(repostats) as repository}
-        <td>
-          {#if verified[repository] && verified[repository].latest}
-            {verified[repository].latest}
-          {/if}
-        </td>
-      {/each}
-    </tr>
-
-    <tr>
-      <th>Differance (human readable)</th>
-      {#each Object.keys(repostats) as repository}
-        <td>
-          {#if verified[repository] && verified[repository].human}
-            {verified[repository].human}
-          {/if}
-        </td>
-      {/each}
-    </tr>
-
-    {#if replicate}
-      <tr>
-        <th>Replication queue length</th>
-        {#each Object.keys(repostats) as repository}
-          <td>
-            {#if replicate[repository]}{replicate[repository]}{/if}
-          </td>
-        {/each}
-      </tr>
-    {/if}
-
-  </table>
-{:else}Loading repository statistics...{/if}
+<table>
+  <tr>
+    <th>Repository</th>
+    {#each reporesponse as repository}
+      <th>{repository.repo}</th>
+    {/each}
+  </tr>
+  <tr>
+    <th>AIP count</th>
+    {#each reporesponse as repository}
+      <td>{repository.aipCount}</td>
+    {/each}
+  </tr>
+  <tr>
+    <th>Total disk used</th>
+    {#each reporesponse as repository}
+      <td>{repository.diskUsed}({filesize(repository.diskUsed)})</td>
+    {/each}
+  </tr>
+  <tr>
+    <th>Earliest validated</th>
+    {#each reporesponse as repository}
+      <td>{new Date(repository.earliest.toString())}</td>
+    {/each}
+  </tr>
+  <tr>
+    <th>Latest validated</th>
+    {#each reporesponse as repository}
+      <td>{new Date(repository.latest.toString())}</td>
+    {/each}
+  </tr>
+  <tr>
+    <th>Difference</th>
+    {#each reporesponse as repository}
+      <td>{repository.difference}</td>
+    {/each}
+  </tr>
+
+  <tr>
+    <th>Replication queue length</th>
+    {#each Object.values(reporesponse) as repository}
+      <td>
+        {#if repository.replicate.content != null}
+          {repository.replicate.content}
+        {:else}{repository.replicate.message}{/if}
+      </td>
+    {/each}
+  </tr>
+
+</table>
```

Comment:
You'll notice that this table is "backwards", in a sense. Populating each row involves looping through the results.

I'd flip this around. Each row should correspond to a repository, and each column should correspond to a field in the data. That way, you only need to `#each` through repositories once.